specification/frontdoor/resource-manager/Microsoft.Network/stable/2020-01-01/frontdoor.json

```diff
@@ -0,0 +1,2292 @@
+        "loadBalancingSettings": {
+          "description": "Load balancing settings associated with this Front Door instance.",
+          "type": "array",
+          "items": {
+            "description": "A load balancing settings",
+            "$ref": "#/definitions/LoadBalancingSettingsModel"
+          }
+        },
+        "healthProbeSettings": {
+          "description": "Health probe settings associated with this Front Door instance.",
+          "type": "array",
+          "items": {
+            "description": "A health probe settings",
+            "$ref": "#/definitions/HealthProbeSettingsModel"
+          }
+        },
+        "backendPools": {
+          "description": "Backend pools available to routing rules.",
+          "type": "array",
+          "items": {
+            "description": "A backend pool",
+            "$ref": "#/definitions/BackendPool"
+          }
+        },
+        "frontendEndpoints": {
+          "description": "Frontend endpoints available to routing rules.",
+          "type": "array",
+          "items": {
+            "description": "A frontend endpoint",
+            "$ref": "#/definitions/FrontendEndpoint"
+          }
+        },
+        "backendPoolsSettings": {
+          "description": "Settings for all backendPools",
+          "$ref": "#/definitions/BackendPoolsSettings"
+        },
+        "enabledState": {
+          "description": "Operational status of the Front Door load balancer. Permitted values are 'Enabled' or 'Disabled'",
+          "enum": [
+            "Enabled",
+            "Disabled"
+          ],
+          "type": "string",
+          "x-ms-enum": {
+            "name": "FrontDoorEnabledState",
+            "modelAsString": true
+          }
+        }
+      }
+    },
+    "FrontDoorProperties": {
+      "description": "The JSON object that contains the properties required to create an endpoint.",
+      "allOf": [
+        {
+          "$ref": "#/definitions/FrontDoorUpdateParameters"
+        }
+      ],
+      "properties": {
+        "resourceState": {
+          "description": "Resource status of the Front Door.",
+          "$ref": "#/definitions/ResourceState"
+        },
+        "provisioningState": {
+          "readOnly": true,
+          "type": "string",
+          "description": "Provisioning state of the Front Door."
+        },
+        "cname": {
+          "readOnly": true,
+          "type": "string",
+          "description": "The host that each frontendEndpoint must CNAME to."
+        },
+        "frontdoorId": {
+          "readOnly": true,
+          "type": "string",
+          "description": "The Id of the frontdoor."
+        },
+        "rulesEngines": {
+          "readOnly": true,
+          "description": "Rules Engine Configurations available to routing rules.",
+          "type": "array",
+          "items": {
+            "description": "A rules engine configuration.",
+            "$ref": "#/definitions/RulesEngine"
+          }
```

Comment:
RP need to ensure ignore retain these properties (if resource has them ) when older clients send request (without these)

Reply:
Yes, RP handles to ignore the fields on the older version and newer version(for create frontdoor). Note that these fields are returned only for GET frontdoor.